.github/workflows/prod.yml

```
@@ -0,0 +1,54 @@
+name: Build Listo, Push to Dockerhub and Deploy to Heroku
+
+on: 
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+env:
+  IMAGE_NAME: listoproject/listo
+  HEROKU_APP_NAME: listo-demo
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build image
+      run: docker build . --file Dockerfile --tag $IMAGE_NAME
+    - name: Log into registry
+      run: echo "${{ secrets.DOCKER_API_KEY }}" | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
```

```suggestion
      run: docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin <<< "${{ secrets.DOCKER_API_KEY }}"
```

Not at all important, but you really don't need `echo` in 99% of cases

---

I gotta remember that one for myself